include/scrnintstr.h

```diff
@@ -620,6 +620,9 @@ typedef struct _Screen {
     SetScreenPixmapProcPtr SetScreenPixmap;
     NameWindowPixmapProcPtr NameWindowPixmap;
 
+    /* This field is used by the 470 proprietary nvidia DDX driver, and should always be NULL */
+    PixmapPtr pScratchPixmap;   /* scratch pixmap "pool" (unused / NULL in modern servers) */
```

Comment:
How does that affect the other drivers ?
Is this an ABI change (related to current 25.0.x) ?